### PR TITLE
Resource list

### PR DESCRIPTION
**Pull request by RabiaSajjad**
- display resource list when creating new resource
- display language(s) and format within the resource list

### CODE REVIEW

File: ckanext/canada/templates/internal/package/new_resource_not_draft.html [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {% ckan_extends %}
 
-{% block subtitle %}
+{% block secondary_content %}
+ {% snippet 'package/snippets/resources.html', pkg=pkg %}
```
Comment (wardi) [inline ckanext/canada/templates/internal/package/new_resource_not_draft.html:4]: I think this one-line addition is worth sending upstream too. https://github.com/ckan/ckan/blob/f8865626012770a2e67d4caf93cae7dbd1e4f094/ckan/templates/package/new_resource_not_draft.html#L20

File: ckanext/canada/templates/internal/package/snippets/resources.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+{% set can_edit = pkg and h.check_access('package_update', {'id':pkg.id }) %}
+{% if pkg and resources %}
+  {% block resources %}
+    <section class="panel panel-default resources-side-section">
+      {% block resources_inner %}
+        {% block resources_title %}
+          <div class="panel-heading"><i class="fa fa-bars"></i> {{ _("Resources") }}</div>
+        {% endblock %}
+        <div class="panel-body resources-side-panel">
+          {% block resources_list %}
```
Comment (wardi) [inline ckanext/canada/templates/internal/package/snippets/resources.html:11]: Can you `{% ckan_extends %}` and only override this block instead?

Comment (RabiaSajjad) [reply ckanext/canada/templates/internal/package/snippets/resources.html]: the `resource`s block needs to be overwritten as we are displaying it using panel classes

**Review by wardi — APPROVED** [review]

File: ckanext/canada/templates/internal/package/new_resource_not_draft.html (hunk 1 above)
Comment (wardi) [reply ckanext/canada/templates/internal/package/new_resource_not_draft.html:4]: done here https://github.com/ckan/ckan/pull/7889

**Pull request merged**